datalad/distribution/publish.py

```diff
@@ -104,17 +100,24 @@ class Publish(Interface):
                 "to subdatasets in case `recursive` is also given.",
             constraints=EnsureStr() | EnsureNone(),
             nargs='*'),
-        annex_copy_opts=Parameter(
-            args=("--annex-copy-opts",),
-            metavar='OPT_STR',  # better name?
-            doc="options passed to 'annex copy'",
-            constraints=EnsureStr() | EnsureNone(),))
+        recursive=recursion_flag,
+        recursion_limit=recursion_limit,
+        git_opts=git_opts,
+        annex_opts=annex_opts,
+        annex_copy_opts=annex_copy_opts
```

Comment:
we would need something like "--skip-failing".  or "--failing=(fail,skip)" (may be also "retry" but not sure).
The thing is that e.g. for openfmri I have some subdatasets which I did initialize, and they are part of the superdataset but not really ready for publication. so their pushing might fail.  Sure I could cleanse it all up, but I think allowing for skipping failing might be a good usability feature to have